hiera.yaml

```diff
@@ -6,9 +6,13 @@ defaults:  # Used for any hierarchy level that omits these keys.
   data_hash: yaml_data  # Use the built-in YAML backend.
 
 hierarchy:
-  - name: 'Full Version'
+  - name: 'Operating System Full Version'
+    path: '%{facts.os.name}-%{facts.os.release.full}.yaml'
+  - name: 'Operating System Major Version'
+    path: '%{facts.os.name}-%{facts.os.release.major}.yaml'
+  - name: 'Operating System Family Full Version'
     path: '%{facts.os.family}-%{facts.os.release.full}.yaml'
-  - name: 'Major Version'
+  - name: 'Operating System Family Major Version'
```

Comment:
I'm a bit concerned about these changes, as os.name and os.family might collide, e.g. Debian is used as name and family as well. However, on the OSes claimed to be supported acceptance checks are succeeding.

Reply:
The logic was good with CentOS / RedHat since on CentOS, os.name=CentOS and os.family=CentOS, and CentOS 7 is basically RedHat 7; CentOS 8 is basically RedHat 8 etc.

For Debian vs. Ubuntu, the versions do not match, so I had to be more specific.  I think this is the minimal change required to cope with Debian 10 and Ubuntu 20.04.